.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,85 @@
+name: "Build, push to GHCR and deploy image to GKE cluster"
+
+on:
+  push:
+    tags: [ '*' ]
+
+env:
+  RELEASE_TAG: ${{ github.ref_name }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        id: buildx
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
```

Comment:
Not sure what the point is here. I think the norm would be to use `${{ github.actor }}`. Why authenticate as the repo owner?

Reply:
Not my work -- taken from the original workflow. 
Figured my timeframe was limited to a certain scope - so didn't fix this in this iteration.